src/uwtools/tests/test_cli.py

```diff
@@ -192,17 +192,17 @@ def test__dispatch_config(params):
     submode, funcname = params
     args = ns()
     vars(args).update({STR.submode: submode})
-    with patch.object(cli, funcname) as m:
+    with patch.object(cli, funcname) as module:
         cli._dispatch_config(args)
-    assert m.called_once_with(args)
+    assert module.called_once_with(args)
 
 
 def test__dispatch_config_compare():
     args = ns()
     vars(args).update({STR.file1path: 1, STR.file1fmt: 2, STR.file2path: 3, STR.file2fmt: 4})
-    with patch.object(cli.uwtools.config.core, "compare_configs") as m:
+    with patch.object(cli.uwtools.config.core, "compare_configs") as module:
```

Comment:
Typically, where the name of the object being mocked is known, we've just used that name as the name of the mock, so I'd suggest `... as compare_configs` in this case.